speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/SQLDatabaseStorageTests.cpp

```diff
@@ -4,46 +4,51 @@
 #include "SQLDatabaseStorage.h"
 #include "../rppUtils/RppUtils.hpp"
 
-/// @brief Creates a database
-#define GET_CREATED_DATABASE_PATH \
-    std::string SimulationDataPath("SimulationData\\");\
-    ASSERT_TRUE(FindFileDirectory(SimulationDataPath, p_fileName));\
-    std::ifstream inputFile(SimulationDataPath + p_fileName);\
-    std::string reading;\
-    ASSERT_TRUE(inputFile.good());\
-    inputFile.clear();\
-    inputFile.seekg(0, std::ios::beg);\
-    inputFile >> reading;\
-    const std::string databasePath = SimulationDataPath + "Simulation_" + reading + "_database.sqlite3";
-
-/// @brief             Tests a database creation fail
-/// @param  p_fileName The filename
-void FailToCreateDatabase(const std::string& p_fileName)
+/// @brief connects to a database and inserts test data. Then closes database
+/// @param p_password password to connect to database
+/// @param p_inputFile test data to insert
+void OpenStoreCloseDatabase(const std::string& p_password, const char* p_inputFile)
```

Comment:
Bit of a vague name for a function, InsertData or something similar would be more clear